bin/zk

```diff
@@ -1,5 +1,37 @@
 #!/usr/bin/env bash
 
+# checks that the current directory that user is in, is inside the $ZKSYNC_HOME.
```

Comment:
Maybe it's better to move these checks to the `zk` itself?
I think introducing the split of logic between the `zk` itself and shell script invoking it may be confusing.

Reply:
I've originally started putting it in zk itself (as writing typescript is a lot easier than shell ;-)

But the problem is, that zk is called 'too late'. For example, if you're using a wrong version of yarn, you'll get all the errors - and you'll not see the message.

similar for directory - all the 'yarn compile' work will run - and then you'll have to re-do it again.

Reply:
Makes sense. This then will make a good comment in the code!